src/ompl/tools/thunder/src/Thunder.cpp

```diff
@@ -301,7 +319,7 @@ ompl::base::PlannerStatus ompl::tools::Thunder::solve(const base::PlannerTermina
                 log.is_saved = "less_2_states";
                 log.too_short = true;
             }
-            else if (false)  // always add when from recall
+            else if (true)  // always add when from recall
```

Comment:
Should we make this into a flag?

Reply:
Is there a performance difference/use for controlling this? I haven't experimented with the flag false vs true

Reply:
Yes, I can write a unit test for this. It needs to be true when we are building dat files most of the time. When running on robot this part of the code does not even run, so it does not matter. The only time we want it to be false is if we are building dat files quickly for some reason.

Reply:
This can be kept as true for now, I have a flag for it in my PR, and I will worry about the unit test as soon as we are done with the more pressing things

Reply:
added a todo for you, post this PR